tests/sentry/api/endpoints/test_project_rules.py

```diff
@@ -480,69 +457,71 @@ def test_comparison_condition_validation(self):
             == "Select a valid choice. bad data is not one of the available choices."
         )
 
-    @patch("sentry.mediators.alert_rule_actions.AlertRuleActionCreator.run")
-    def test_runs_alert_rule_action_creator(self, mock_alert_rule_action_creator):
```

Comment:
Did this get replaced by other tests?

Reply:
Yeah I wrote that test 7 months ago before I understood the `responses` library. It makes more sense to test the output from the entire endpoint than to test that some function got the arguments it expected, so I opted to remove it :+1:

Reply:
The 'output from entire endpoint' point I mention is what the two new tests in each file are focused on (i.e. surfacing errors and creating/editing the rules)

Reply:
Just checking, it's a beefy diff and I wasn't sure if I missed it being moved!